index.html

```diff
@@ -3,7 +3,31 @@
 	<head>
 		<title>Introduction to the DOM</title>
 		<link rel="stylesheet" href="./styles.css">
+		<script defer="defer" src="components/Dropdown/dropdown.js"></script>
 	</head>
 	<body>
+		<div class="header">
+			<div class="header__dropdown header_dropdown--hover">
+      				<div class="dropdown__title">Dropdown</div>
+      					<div class="dropdown__list">
+							<a href="#" >Lambda</a>
+							<a href="#" >Google</a>
+							<a href="#" >MDN</a>
+      				</div>
+			</div>
+				<h3 class="header__headline">Header</h3>
+		</div>
+			<div class="boxes">
+				<div class="boxes__leftBox">
+					<h3 class="leftBox__headline">Placeholder</h3>
+					<p class="leftBox__content">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce risus nibh, gravida nec felis quis, facilisis facilisis lectus. Nulla ac orci pretium, condimentum orci quis, accumsan nisi. Aliquam erat volutpat. Curabitur cursus mattis libero, at viverra risus hendrerit quis. Fusce imperdiet tristique tortor non tincidunt. Mauris accumsan urna nec augue feugiat porta. Proin vitae magna in ex malesuada laoreet eget a nulla. Aliquam tristique et elit at consequat. In hac habitasse platea dictumst.
+					</p>
+				</div>
+				<div class="boxes__rightBox">
+					<h3 class="rightBox__headline">Placeholder</h3>
```

Comment:
You can also write this another way as `boxes__rightBox--headline`. This is just for organization. The standard is that if you are going to nest it will look something like `item1__item2--item3`. The reason why I would recommend you adding the `boxes` to the front is for clarification when you do your CSS or DOM. You will be able to clearly see where that `rightBox__headline` is from!